saleor/static/scss/layouts/_checkout.scss

```diff
@@ -91,8 +91,8 @@
           border-radius: 12px;
         }
         .btn {
-          height: 3.6rem;
-          padding: $global-padding;
+          height: calc(2.25rem + 2px);
```

Comment:
This change is however fine; after the upgrade the voucher input is narrower and looks better.